physical/mysql_test.go

```diff
@@ -0,0 +1,86 @@
+package physical
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func TestMySQLBackend(t *testing.T) {
```

Comment:
Once you create the backend you should use the test methods here: 
https://github.com/hashicorp/vault/blob/master/physical/physical_test.go#L26
https://github.com/hashicorp/vault/blob/master/physical/physical_test.go#L130

Those are used to verify the behavior works as Vault expects